anago

```diff
@@ -920,6 +920,13 @@ if [[ -n "$PARENT_BRANCH" ]]; then
   logecho "$ATTENTION: $RELEASE_BRANCH is a NEW branch off $PARENT_BRANCH!"
 fi
 
+if ! ((FLAGS_nomock)); then
+  logecho
+  logecho "$ATTENTION: This is a mock (--mock) run." \
+          "Publishing will be based on the above values for" \
```

Comment:
That is mostly the case, though with the recently added --gcrio_repo= flag, a user does have the ability to run a mock build but point to a different gcrio repo.  Still, that would fail to run and push unless permission allows.

The pause at SESSION VALUES was always meant to be a "READ THIS FIRST" which is why there is a prompt there to continue after verifying the values above, but this PR adds an additional level of warning about running in mock mode.